css/sass-lint/readme.md

```diff
@@ -0,0 +1,55 @@
+# Sass-Lint
+
+Sass-Lint is a pure node.js-only tool to help you write clean and consistent SCSS and Sass. You can run it manually from the command line, which will output any problems with your SCSS, including the filename and line number(s). Newly generated Adaptive.js projects use this tool by default, leveraging a Grunt task wrapper to invoke it.
+
+When enabled in an Adaptive.js project, Sass files may be linted by invoking `grunt sasslint` or `grunt scsslint`.
```

Comment:
Well the Grunt wrapper uses `grunt sasslint` as the command, but I've also aliased `grunt scsslint` to it as well (they both do the exact same thing) to maintain backwards compatibility with naming conventions. 

I'm open to only having one task here to avoid any ambiguity.